tests/ui/test_ui_tools.py

```diff
@@ -1302,13 +1302,21 @@ def test_keypress_edit_message(self, mocker, message_fixture,
             <p>C</p>""", "░ A\n░ B\n\nC"),
         ("""<blockquote>
                 <p><a href='https://chat.zulip.org/'</a>czo</p>
-            </blockquote>""", "░ [czo](https://chat.zulip.org/)\n")
+            </blockquote>""", "░ [czo](https://chat.zulip.org/)\n"),
+        pytest.param("""<blockquote>
+                            <blockquote>
+                                <p>A<br>
+                                B</p>
+                            </blockquote>
+                        </blockquote>
+            """, "░ ░ A\n░ ░ B", marks=pytest.mark.xfail(reason="nested bq"))
     ], ids=[
         "quoted level 1",
         "quoted level 2",
         "quoted level 3",
         "multi-line quoting",
-        "quoting with links"
+        "quoting with links",
+        "nested blockquote"
```

Comment:
Is this not more like "quoted multiline level 2"?  (though we could likely skip the 'quoted' in the ids)

Reply:
Agreed. That is a better description